README.md

```diff
@@ -1 +1,966 @@
+  
+```
+   $ballerina build 
+  
+   ./target/phone_store.balx
+        @docker                  - complete 3/3 
+
+        Run following command to start docker container:
+        docker run -d -p 9090:9090 ballerina.guides.io/phone_store_service:v1.0
+
+    ./target/phone_order_delivery_service.balx
+        @docker                  - complete 3/3 
+
+        Run following command to start docker container:
+        docker run -d -p 9091:9091 ballerina.guides.io/phone_order_delivery_service:v1.0
+
+```
+
+- Once you successfully build the Docker image, you can run it with the `` docker run`` command that is shown in the previous step.  
+
+```bash
+  docker run -d -p 9090:9090 ballerina.guides.io/phone_store_service:v1.0
+  docker run -d -p 9091:9091 ballerina.guides.io/phone_order_delivery_service:v1.0  
+```
+
+   Here we run the Docker image with flag`` -p <host_port>:<container_port>`` so that we use the host port 9090 and the container port 9090. Therefore you can access the service through the host port. 
+
+- Verify docker container is running with the use of `` $ docker ps``. The status of the Docker container should be shown as 'Up'. 
+
+- You can access the service using the same curl commands that we've used above.
+```bash
+    curl -v -X POST -d \
+   '{"Name":"John", "Address":"20, Palm Grove, Colombo, Sri Lanka", 
+   "ContactNumber":"+94718930874", "PhoneName":"Apple:190000"}' \
+   "http://localhost:9090/phonestore/placeOrder" -H "Content-Type:application/json"
+```
+### Deploying on Kubernetes
+
+- You can run the service that we developed above, on Kubernetes. The Ballerina language offers native support to run a Ballerina program on Kubernetes, with the use of Kubernetes annotations that you can include as part of your 
+service code. Also, it will take care of the creation of the Docker images. So you don't need to explicitly create Docker images prior to deploying it on Kubernetes. Refer [Ballerina_Kubernetes_Extension](https://github.com/ballerinax/kubernetes) for more details and samples on Kubernetes deployment with Ballerina. You can also find details on using Minikube to deploy Ballerina programs. 
+
+- Since this guide requires `ActiveMQ` as a prerequisite, you need an additional step to create a pod for `ActiveMQ` and use it with our sample.  
+
+- Navigate to `message_construction_patterns/resources` directory and run the below command to create the ActiveMQ pod by creating a deployment and service for ActiveMQ. You can find the deployment descriptor and service descriptor in the `./resources/kubernetes` folder.
+
+```bash
+   $ kubectl create -f ./kubernetes/
+```
+
+- Now let's see how we can deploy the `phone_store_service` on Kubernetes. We need to import `` ballerinax/kubernetes; `` and use ``@kubernetes``annotations as shown below to enable kubernetes deployment.
```

Comment:
Remove the semi colon. Also, no need for double back ticks right?

Leave a space between "@kubernetes" and "annotations".

kubernetes -> Kubernetes. Check and fix in all relevant places